src/docs/menu.js

```diff
@@ -0,0 +1,101 @@
+const M = (title, path, subMenu, indented = false) => ({
+    title,
+    path: '/docs/' + (path ? path + '/' : ''),
+    subMenu,
+    indented
+})
+
+export const MENU = [
+    {
+        title: 'Architecture'
+    },
+    M(
+        'Overview',
+        'architecture'
+    ),
+    M(
+        'Extensions',
+        'extensions'
+    ),
+    M(
+        'Services and Contributions',
+        'services_and_contributions'
+    ),
+    {
+        title: 'Using Theia'
+    },
+    M(
+        'Build your own IDE',
+        'composing_applications'
+    ),
+    M(
+        'Authoring Theia Extensions',
```

Comment:
Nit: I found the previous title a bit nicer (shorter):

```suggestion
        'Authoring an Extension',
```